cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NamespaceHttpHandler.java

```diff
@@ -67,64 +66,42 @@ public void getAllNamespaces(HttpRequest request, HttpResponder responder) throw
   @GET
   @Path("/namespaces/{namespace-id}")
   public void getNamespace(HttpRequest request, HttpResponder responder,
-                           @PathParam("namespace-id") String namespaceId) {
-    try {
-      NamespaceMeta ns = namespaceAdmin.get(Id.Namespace.from(namespaceId));
-      responder.sendJson(HttpResponseStatus.OK, ns);
-    } catch (NotFoundException e) {
-      responder.sendString(HttpResponseStatus.NOT_FOUND, String.format("Namespace %s not found", namespaceId));
-    } catch (Exception e) {
-      LOG.error("Internal error while getting namespace '{}'", namespaceId, e);
-      responder.sendString(HttpResponseStatus.INTERNAL_SERVER_ERROR, e.getMessage());
-    }
+                           @PathParam("namespace-id") String namespaceId) throws Exception {
+    NamespaceMeta ns = namespaceAdmin.get(Id.Namespace.from(namespaceId));
+    responder.sendJson(HttpResponseStatus.OK, ns);
   }
 
 
   @PUT
   @Path("/namespaces/{namespace-id}/properties")
   public void updateNamespaceProperties(HttpRequest request, HttpResponder responder,
-                                        @PathParam("namespace-id") String namespaceId) {
-    try {
-      NamespaceMeta meta = parseBody(request, NamespaceMeta.class);
-      namespaceAdmin.updateProperties(Id.Namespace.from(namespaceId), meta);
-      responder.sendString(HttpResponseStatus.OK, "Properties updated");
-    } catch (NotFoundException e) {
-      responder.sendString(HttpResponseStatus.NOT_FOUND, String.format("Namespace %s not found", namespaceId));
-    } catch (Exception e) {
-      LOG.error("Failed to read namespace metadata request body.", e);
-      responder.sendString(HttpResponseStatus.INTERNAL_SERVER_ERROR, e.getMessage());
-    }
+                                        @PathParam("namespace-id") String namespaceId) throws Exception {
+    NamespaceMeta meta = parseBody(request, NamespaceMeta.class);
+    namespaceAdmin.updateProperties(Id.Namespace.from(namespaceId), meta);
+    responder.sendString(HttpResponseStatus.OK, "Properties updated");
```

Comment:
nit: Not your PR, but should we send a more meaningful response here: ``Properties for namespace %s updated``?